### PR TITLE
Wrote summary to every class as well as removed unnessesary usings

### PR DESCRIPTION
**Pull request by annahj1006**
Just added the summary on top of each class, they are vaguely descrptive, since its a summary of the whole class

### CODE REVIEW

File: src/Chirp.Core/IAuthorRepository.cs [hunk 1]
```diff
@@ -1,3 +1,10 @@
+/*
+<Summary>
+This is the author Repository interface
+The Author Repository will inherit from this interface, and implement the methods.
```
Comment (SkarpKat) [inline src/Chirp.Core/IAuthorRepository.cs:4]: I know I'm annoying but it's not inherit but implement when it's an interface.

File: src/Chirp.Core/ICheepRepository.cs [hunk 2]
```diff
@@ -1,3 +1,10 @@
+/*
+<Summary>
+This is the Cheep Repository interface
+The Cheep Repository will inherit from this interface, and implement the methods.
```
Comment (SkarpKat) [inline src/Chirp.Core/ICheepRepository.cs:4]: I know I'm annoying but it's not inherit but implement when it's an interface.

File: src/Chirp.Core/ICheepService.cs [hunk 3]
```diff
@@ -1,5 +1,11 @@
 using Chirp.Core;
 
+/*
+<Summary>
+This is the Cheep Service interface
+The Cheep Service will inherit from this interface, and implement the methods.
```
Comment (SkarpKat) [inline src/Chirp.Core/ICheepService.cs:6]: I know I'm annoying but it's not inherit but implement when it's an interface.

**Review by SkarpKat — CHANGES_REQUESTED** [review]

File: src/Chirp.Razor/Pages/Public.cshtml.cs [hunk 4]
```diff
@@ -1,7 +1,14 @@
 ﻿using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 using Chirp.Core;
-using Chirp.Infrastructure;
+
+/*
+<Summary>
+This is model for the public page
```
Comment (Lukan707) [inline src/Chirp.Razor/Pages/Public.cshtml.cs:7]: I believe there is missing a 'the' on line 7.

Maybe also add that the razor pages links it with the Public.cshtml file

File: src/Chirp.Razor/Pages/UserPage.cshtml.cs [hunk 5]
```diff
@@ -3,6 +3,13 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
+/*
+<Summary>
+This is the User Pages model
+The user page is where the user can see their own information, so this is used for the forget me feature.
+</Summary>
```
Comment (Lukan707) [inline src/Chirp.Razor/Pages/UserPage.cshtml.cs:10]: Again maybe add that is it linked with razor pages by UserPage.cshtml

File: src/Chirp.Razor/Pages/UserTimeline.cshtml.cs [hunk 6]
```diff
@@ -1,10 +1,16 @@
...
-using Chirp.Infrastructure;
 using Chirp.Core;
 using Microsoft.AspNetCore.Authorization;
 namespace Chirp.Razor.Pages;
 
+/*
+<Summary>
+This is the users timeline model
+This is where the user can see their own timeline and those they follow.
+</Summary>
```
Comment (Lukan707) [inline src/Chirp.Razor/Pages/UserTimeline.cshtml.cs:11]: And again again :) , maybe add that razor pages links this with the UserTimeline.cshtml file.

**Review by Lukan707 — CHANGES_REQUESTED** [review]
Nice work :)

**Review by SkarpKat — APPROVED** [review]

**Review by Lukan707 — APPROVED** [review]
Great :)

**Pull request merged**